TestProjects/UniversalGraphicsTest_Terrain/Assets/CommonAssets/ZeroVariantColorTextureAlphaOnly.shader

```diff
@@ -0,0 +1,125 @@
+Shader "Universal Render Pipeline/Test/Zero Variant Color Texture Alpha"
+{
+    Properties
+    {
+        [MainTexture] _BaseMap("Texture", 2D) = "white" {}
+        [MainColor] _BaseColor("Color", Color) = (1, 1, 1, 1)
+        _Cutoff("AlphaCutout", Range(0.0, 1.0)) = 0.0
+
+        // BlendMode
+        _Surface("__surface", Float) = 0.0
+        _Blend("__mode", Float) = 0.0
+        _Cull("__cull", Float) = 2.0
+        [HideInInspector] _AlphaClip("__clip", Float) = 1.0
+        [HideInInspector] _BlendOp("__blendop", Float) = 0.0
+        [HideInInspector] _SrcBlend("__src", Float) = 1.0
+        [HideInInspector] _DstBlend("__dst", Float) = 0.0
+        [HideInInspector] _ZWrite("__zw", Float) = 1.0
+
+        // Editmode props
+        _QueueOffset("Queue offset", Float) = 0.0
+    }
+
+    HLSLINCLUDE
+        #include "Packages/com.unity.render-pipelines.universal/Shaders/UnlitInput.hlsl"
+        #pragma target 2.0
+
+        struct Attributes
+        {
+            float4 positionOS : POSITION;
+            float2 uv : TEXCOORD0;
+
+            UNITY_VERTEX_INPUT_INSTANCE_ID
+        };
+
+        struct Varyings
+        {
+            float4 positionCS : SV_POSITION;
+            float2 uv : TEXCOORD0;
+
+            UNITY_VERTEX_INPUT_INSTANCE_ID
+            UNITY_VERTEX_OUTPUT_STEREO
+        };
+
+        Varyings Vert(Attributes input)
+        {
+            Varyings output = (Varyings)0;
+
+            UNITY_SETUP_INSTANCE_ID(input);
+            UNITY_TRANSFER_INSTANCE_ID(input, output);
+            UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(output);
+
+            VertexPositionInputs vertexInput = GetVertexPositionInputs(input.positionOS.xyz);
+
+            output.positionCS = vertexInput.positionCS;
+            output.uv = TRANSFORM_TEX(input.uv, _BaseMap);
+            return output;
+        }
+
+        half4 Frag(Varyings input) : SV_Target
+        {
+            UNITY_SETUP_INSTANCE_ID(input);
+            UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
+
+            half2 uv = input.uv;
+            half4 texColor = SAMPLE_TEXTURE2D(_BaseMap, sampler_BaseMap, uv);
+            half3 color = texColor.rgb * _BaseColor.rgb;
+            half alpha = texColor.a * _BaseColor.a;
+            clip(alpha - _Cutoff);
+            half4 finalColor = half4(color, alpha);
+            return finalColor;
+            //return half4(alpha.xxx, 1);
+        }
+
+    ENDHLSL
+
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "IgnoreProjector" = "True" "RenderPipeline" = "UniversalPipeline" "ShaderModel"="2.0"}
+        LOD 100
+
+        Blend [_SrcBlend][_DstBlend]
+        ZWrite [_ZWrite]
+        Cull [_Cull]
+
+        Pass
+        {
+            Name "ZeroVariantColorTexAlphaOnly"
+
+            HLSLPROGRAM
+                #pragma vertex Vert
+                #pragma fragment Frag
+            ENDHLSL
+        }
+
+        /*Pass
```

Comment:
Should these commented out sections just be removed?

Reply:
Yes. I will remove those.